lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -1713,8 +1749,10 @@ namespace Js
 
         builtinFuncs[BuiltinFunction::JavascriptArray_Unshift]            = library->AddFunctionToLibraryObject(arrayPrototype, PropertyIds::unshift,         &JavascriptArray::EntryInfo::Unshift,           1);
 
-
-        builtinFuncs[BuiltinFunction::JavascriptArray_IndexOf]        = library->AddFunctionToLibraryObject(arrayPrototype, PropertyIds::indexOf,         &JavascriptArray::EntryInfo::IndexOf,           1);
+        if (!scriptContext->IsJsBuiltInEnabled())
```

Comment:
you can remove the scriptContext function for this and check `CONFIG_FLAG(JsBuiltin)` instead